k8s/base.py

```diff
@@ -184,7 +184,7 @@ def as_dict(self):
                 d[_api_name(field.name)] = value
         return d
 
-    def update(self, other):
+    def merge(self, other):
```

Comment:
Maybe we should keep `update` (pointing to `merge`) for backwards compatibility?

Reply:
> Maybe we should keep update (pointing to merge) for backwards compatibility?

Agreed.